WordPressKit/HTTPClient.swift

```diff
@@ -189,3 +192,14 @@ extension WordPressAPIResult {
     }
 
 }
+
+extension Progress {
+    func update(totoalUnit: Int64, with progress: Progress, queue: DispatchQueue) -> AnyCancellable {
```

Comment:
```suggestion
    func update(totalUnit: Int64, with progress: Progress, queue: DispatchQueue) -> AnyCancellable {
```

...then required additional adjustments